lib/cache-save.js

```diff
@@ -1,9 +1,16 @@
 import * as core from "@actions/core";
 import * as cache from "@actions/cache";
 
-import { getCacheDirectory, State } from "./cache-utils";
+import { getCacheDirectory, isCacheEnabled, State } from "./cache-utils";
 
 async function saveCache() {
+  const cacheInput = core.getInput("cache");
```

Comment:
Maybe we should just use `core.getBooleanInput("cache")`?

Currently, some values (`"True", "TRUE"`) that are [considered truthy](https://yaml.org/spec/1.2.2/#1022-tag-resolution) might be result in `isCacheEnabled = false`.